makedist.pl

```diff
@@ -84,8 +88,17 @@
 		@mlist = @mlist_excluded;
 		}
 	}
-@dirlist = ( "vendor_perl" );
 
+# Build EOL data
+if (-r "$root_directory/webmin/os-eol-lib.pl") {
+	print "Building OS EOL data\n";
+	do "$root_directory/web-lib-funcs.pl";
+	do "$root_directory/webmin/os-eol-lib.pl";
+	&eol_build_all_os_data("$root_directory/os_eol.json");
```

Comment:
What does this function do other than just downloading the JSON file? Could we replace it with just a `wget` command?

Reply:
Not really. We need to parse JSON as well.

But we can remove it and run it manually or in pre-build scripts.

Reply:
Does the JSON get converted into some other format? It wasn't clear to me what that function is doing..

Reply:
> Does the JSON get converted into some other format? It wasn't clear to me what that function is doing..

It's converted to the JSON string.

Reply:
But isn't what's downloaded from the API already in JSON format?

Reply:
> But isn't what's downloaded from the API already in JSON format?

It is, but we need to combine it for multiple different OSes.

Reply:
Ah, I see ... ok that makes sense.